react/index.js

```diff
@@ -7,6 +7,7 @@ export { default as Rating } from './Rating/Rating';
 export { default as Header } from './Header/Header';
 export { default as Footer } from './Footer/Footer';
 export { default as Loader } from './Loader/Loader';
+export { default as DisableScrollOutside } from './DisableScrollOutside/DisableScrollOutside';
```

Comment:
Not sure this should be exported, seems more like a utility than a company specific style.